javascript-review.js

```diff
@@ -1,115 +1,240 @@
+// import { url } from "inspector";
+
+const fetch = require('node-fetch');
+
 // You may NOT use Google. These questions are to test how you are progressing through the material, and to give you some feedback about where you might improve. Get through as many of these as you can. If you don’t get through all of them it’s ok - the test is designed to be difficult. If you can’t remember a bit of code, write the pseudo-code that represents how you would write your JS code.
 
 // 1.
 
 // Create an object that has four properties. One property should be set so that the value is a number, another property set to a string, the third to an array, and the fourth to a function. This function should simply console.log the value of the first property.
 
+
+const clinton = {
+    name: "Clinton",
+    age: 32,
+    hobbies: ["soccer", "guitar", "coding", "beer"],
+    function: function logFirst() {console.log(this.name)}
+};
+
+console.log(clinton)
+clinton.function()
```

Comment:
✅ 
maybe for next time (to avoid weird bugs) don't call the key `function` use another name